fe/fe-core/src/test/java/com/starrocks/sql/plan/PlanTestNoneDBBase.java

```diff
@@ -88,6 +88,7 @@ public static void beforeClass() throws Exception {
         connectContext = UtFrameUtils.createDefaultCtx();
         starRocksAssert = new StarRocksAssert(connectContext);
         connectContext.getSessionVariable().setOptimizerExecuteTimeout(30000);
+        connectContext.getSessionVariable().setUseLowCardinalityOptimizeV2(false);
```

Comment:
could turn on the v2 version to test all these cases?

Reply:
ok, but unnecessary, the low-cardinality V1 only do test in his case too